packages/components/psammead-story-promo/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 4.0.0-alpha.2 | [PR#2989](https://github.com/bbc/psammead/pull/2989) Update screenshot |
```

Comment:
```suggestion
| 4.0.0-alpha.2 | [PR#2989](https://github.com/bbc/psammead/pull/2989) Update snapshot |
```

Reply:
I'm guessing this should be `snapshot` ?

Reply:
oops 🙈  you're right